File: AR-VR/devAyushDubey_ayushdubey_2024csit1122_2/FunctionAndArrays/BrokenEconomy.cpp
#include<iostream>
using namespace std;

int main(){
    int n,i;
    cin>>n;
    int* arr = new int[n];
    for(int i = 0 ; i < n; i++){
        cin>>arr[i];
    }
    int data;
    cin>>data;
    for(i=0;i<n;i++){
        if(data==arr[i]){
            cout<<arr[i];
            return 0;
        }
    }
    i=0;
    while(data>arr[i]){
        i++;
    }
    if(i==n || i==0)
        cout<<arr[i];
    else
        cout<<arr[i]<<endl<<arr[i-1]<<endl;
        
    return 0;
    // write your code here
    
}

File: IOT/anshulgurjar53_AnshulSinghGurjar_2024it1193_2/FunctionAndArrays/Brokeneconomy.java
import java.io.*;
import java.util.*;

public class Main{

public static void main(String[] args) throws Exception {
    // write your code here
    Scanner sc= new Scanner(System.in);
    int n= sc.nextInt();
    int a[]=new int[n];
    for(int i=0;i<n;i++)
    a[i]=sc.nextInt();
    int d= sc.nextInt();int i;
    for(i=0;i<n;i++)
    {
        if(a[i]==d){
            System.out.println(d);break;
        }
        else if(a[i]>d)
        break;

    }
    
    
        System.out.println(a[i]);
        System.out.println(a[i-1]);
    
 }

}